extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeCommentsExtractorTest.java

```diff
@@ -328,5 +331,17 @@ public void testGetCommentsFirstReplies() throws IOException, ExtractionExceptio
             assertEquals("First", replies.getItems().get(0).getCommentText(),
                     "First reply comment did not match");
         }
+
+        @Test
+        public void testGetCommentsReplyCount() throws IOException, ExtractionException {
+            final InfoItemsPage<CommentsInfoItem> comments = extractor.getInitialPage();
+
+            DefaultTests.defaultTestListOfItems(YouTube, comments.getItems(), comments.getErrors());
+
+            CommentsInfoItem firstComment = comments.getItems().get(0);
+
+            assertNotEquals(UNKNOWN_REPLY_COUNT, firstComment.getReplyCount(), "First reply comment can't get count");
+            assertGreater(300, firstComment.getReplyCount(), "First reply comment count is smaller than the expected 300 comments");
```

Comment:
I think you should simply use [`assertGreater(long, long)`](https://github.com/TeamNewPipe/NewPipeExtractor/blob/197d8876fcbc8126fb74d1c45563f1fe3bef0bcd/extractor/src/test/java/org/schabi/newpipe/extractor/ExtractorAsserts.java#L77), which will show the message `ACTUAL_COMMENT_REPLY_COUNT  is not > EXPECTED_COMMENT_REPLY_COUNT`.

```suggestion
            assertGreater(300, firstComment.getReplyCount());
```

This has the advantage to know the real count extracted, which is useful to update the expected count if some users remove their comment reply(ies), and so make that the comment reply count is under the expected value.

Reply:
Done. https://github.com/TeamNewPipe/NewPipeExtractor/commit/f88e235089f4dc8d2b260192f1ebd01dd4dee20d